auto_nag/scripts/topcrash_add_keyword.py

```diff
@@ -0,0 +1,175 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+import re
+
+from libmozdata.bugzilla import Bugzilla
+from libmozdata.connection import Connection
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.topcrash import Topcrash
+
+# TODO: should be moved when resolving https://github.com/mozilla/relman-auto-nag/issues/1384
+HIGH_SEVERITY = {"S1", "critical", "S2", "major"}
+
+MAX_SIGNATURES_PER_QUERY = 30
+SIGNATURES_PAT = re.compile(r"\[@ ?(.*?) ?\]")
+
+
+class TopcrashAddKeyword(BzCleaner):
+    def description(self):
+        return "Bugs with topcrash keyword but incorrect severity"
+
+    def columns(self):
+        return ["id", "summary", "severity", "added_keyword"]
+
+    @staticmethod
+    def __get_mail_to_ni(bug):
```

Comment:
I think it could be move into `utils.py` in order to be used in other scripts.

Reply:
I agree, done ee60958cbd6e5c8a76ea195877485d50b6906737. Other tools use the same clone; I will refactor them in separate PR to use `utils` as well.